### DISCUSSION TITLE
this framework support graphql-ws transport protocol

### DISCUSSION THREAD

**Discussion by vincenzopalazzo** [graphql-rust/juniper#1078] [Q&A] [answered]
I'm building a debug server for a client library and I would like to avoid using Typescript and I more this server in rust, but I'm not able to find in the docs if this library support already the graphql-ws transport protocol https://github.com/enisdenjo/graphql-ws

**Comment by tyranron** [comment, marked as answer]
Not yet supported. See https://github.com/graphql-rust/juniper/issues/1022

**Reply by vincenzopalazzo** [reply]
Yep, I will work on it in my free time!
